generate_build_data.py

```diff
@@ -93,23 +128,36 @@ def write_lib(name, data, f):
     ]
 
     f.write(b"pub const %s: Data = Data {\n" % name)
+    f.write(b"    lib: %s,\n" % string_literal(lib))
     write_list(b"sources", map(string_literal, filter(no_platform_sources, data["SOURCES"])), f)
     write_list(b"includes", map(string_literal, data["LOCAL_INCLUDES"]), f)
     write_list(b"defines", defines, f)
     write_list(b"os_libs", map(string_literal, data["OS_LIBS"]), f)
+    write_list(b"use_libs", map(lib_enum, filter(no_zlib, data["USE_LIBS"])), f)
```

Comment:
`no_zlib` is so simple that it's useful to just leave it here:

```suggestion
    write_list(b"use_libs", map(lib_enum, filter(lambda s: "zlib" not in s, data["USE_LIBS"])), f)
```